### PR TITLE
[editorial] First edits 

### PR DESCRIPTION
**Pull request by Sakurann**
Some editorial edits

### CODE REVIEW

File: openid-connect-4-credential-issuance-1_0.md [hunk 1]
```diff
@@ -30,25 +30,25 @@ organization="Microsoft"
 
 .# Abstract
 
-This specification defines an extension of OpenID Connect to allow holders to request issuance and issuers of verifiable credentials issuance of verifiable credentials.
+This specification defines an extension of OpenID Connect to allow holders to request issuance of verifiable credentials.
```
Comment (tlodderstedt) [inline openid-connect-4-credential-issuance-1_0.md]: I think that's to narrow as the protocol certainly also allows to issue the credentials. 

Perhaps: 

"allows issuance of verifiable credentials (in addition to the standard OpenID Connect assertions)."

File: openid-connect-4-credential-issuance-1_0.md [hunk 2]
```diff
@@ -30,25 +30,25 @@ organization="Microsoft"
...
+OpenID Connect is an obvious choice for this use case since it already allows Relying Parties to request identity assertions. Verifiable Credentials are very similar in that they allow an Issuer to assert End-User claims. In contrast to the identity assertions, a verifiable credential follows a pre-defined schema (the credential type) and is bound to a key material allowing the holder to prove the legitimate possession of the credential. This allows direct presentation of the credential without involvment of the credential issuer. This specification caters for those differences. 
 
 # Terminology
 
 Credential
 
-A set of one or more claims made by an issuer. (see [@VC_DATA])
+A set of one or more claims made by an issuer. (see [@VC_DATA]) Note that this definition differs from that in [OpenID].
```
Comment (tlodderstedt) [inline openid-connect-4-credential-issuance-1_0.md:47]: how does the definition differ and why does it matter? We didn't point that out in OIDC4VPs

File: openid-connect-4-credential-issuance-1_0.md [hunk 3]
```diff
@@ -70,23 +70,27 @@ A user comes across an app where she needs to present a credential, e.g. a bank
...
+
+## Deferred Credential Issuance (Asynchronous Credential Issuance?)
 
 The user wants to obtain a digital criminal record certificate. She starts the journey in her wallet and is sent to the issuer service of the responsible government authority. She logs in with her eID and requests the issuance of the certificate. She is being notified that the issuance of the certificate will take a couple of days due to necessary background checks by the authority. She confirms and is sent back to the wallet. The wallet shows a hint in the credential list indicating that issuance of the digital criminal record certificate is under way. A few days later, she receives an e-Mail from the authority that the certificate was sucessfully issued. She visits her wallet, where she is asked after startup whether she wants to download the certificate. She confirms and the new credential is retrieved and stored in her wallet. 
 
 # Requirements
 
-This section describes the requirements this specification aims to fulfill beyond the use cases described above. 
+This section describes the requirements this specification aims to fulfill to realize the use cases described above.
```
Comment (tlodderstedt) [inline openid-connect-4-credential-issuance-1_0.md]: "to fulfill to realize" seems to be a duplicate

File: openid-connect-4-credential-issuance-1_0.md (hunk 2 above)
Comment (Sakurann) [reply openid-connect-4-credential-issuance-1_0.md:47]: Differs in a way that those familiar with OIDF world think of credentials as something that gives access to the user claims such as passwords/biometrics as opposed to the claims itself.
probably we should point out in OIDC4VP too. Unless we want it to be a silent evolution of terminology.

File: openid-connect-4-credential-issuance-1_0.md (hunk 1 above)
Comment (Sakurann) [reply openid-connect-4-credential-issuance-1_0.md]: you're right, you can definitely use this to issue `VCs and OIDC claims together.

File: openid-connect-4-credential-issuance-1_0.md [hunk 4]
```diff
@@ -70,37 +70,41 @@ A user comes across an app where she needs to present a credential, e.g. a bank
...
+
+## Deferred Credential Issuance (Asynchronous Credential Issuance?)
 
 The user wants to obtain a digital criminal record certificate. She starts the journey in her wallet and is sent to the issuer service of the responsible government authority. She logs in with her eID and requests the issuance of the certificate. She is being notified that the issuance of the certificate will take a couple of days due to necessary background checks by the authority. She confirms and is sent back to the wallet. The wallet shows a hint in the credential list indicating that issuance of the digital criminal record certificate is under way. A few days later, she receives an e-Mail from the authority that the certificate was sucessfully issued. She visits her wallet, where she is asked after startup whether she wants to download the certificate. She confirms and the new credential is retrieved and stored in her wallet. 
 
 # Requirements
 
-This section describes the requirements this specification aims to fulfill beyond the use cases described above. 
+This section describes the requirements this specification aims to fulfill the use cases described above.
```
Comment (tlodderstedt) [inline openid-connect-4-credential-issuance-1_0.md]:
```suggestion
This section describes the requirements this specification aims to fulfill beyond the use cases described above. 
```
```suggestion
This section describes the requirements this specification aims to fulfill the use cases described above. 
```

**Pull request merged**